components/dash-core-components/src/components/Tooltip.react.js

```diff
@@ -6,25 +6,35 @@ import _JSXStyle from 'styled-jsx/style'; // eslint-disable-line no-unused-vars
 /**
  * A tooltip with an absolute position.
  */
-const Tooltip = props => {
-    const {bbox, border_color, background_color, id, loading_state} = props;
+const Tooltip = ({
+    show = true,
+    targetable = false,
+    direction = 'right',
+    border_color = '#d6d6d6',
```

Comment:
again, can we (in future?) replace magic color constants with names that are all defined in one place?